validator/testcases/javascript/actions.py

```diff
@@ -939,7 +949,7 @@ def _get_as_num(value):
             return value
         else:
             return int(value)
-    except ValueError, TypeError:
+    except (ValueError, TypeError):
```

Comment:
On Tue, May 05, 2015 at 08:01:30AM -0700, Mathieu Agopian wrote:
>> -    except ValueError, TypeError:
>> +    except (ValueError, TypeError):
>
>:+1:

One of the benefits of skimming through the copious output of 
`flake8`...